src/docs/patterns/forms.mdx

```diff
@@ -82,3 +95,78 @@ In some specific situations, you may need to defer verification or perform it fo
+        '2: On mobile devices, the button is occluded by the native keyboard.',
+    ]}
+/>
+<ImageGrid
+    images={[NextButtonUsageImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={[
+        'On mobile, if there are more than one input fields on the screen, the keyboard button should say "next".',
+        '1: The user entered a valid input. Clicking "next" will autofocus the user to the next text field.',
+        '2: The user entered a invalid input. Clicking "next" will display the error message but will not autofocus to the next text field.'
+    ]}
+/>
+
+### Disable the Submit Button
+
+When there are not a lot of text fields, the submit button can be disabled to indicate that there are some errors in the user input.
+
+<ImageGrid
+    images={[SubmitButtonOnChangeImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={
+        'If you have only one input field, and this field is verified on change, disable the submit button until the user entered the desired input.'
+    }
+/>
+<ImageGrid
+    images={[SubmitButtonAutoSubmitImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={
+        'For the verification code screen, you can auto-submit the code once the user input reaches the desired length (2). After the verification code is recognized (3), the screen pauses for a short moment before auto-transit to the next page. The verify button is preserved for consistency.'
+    }
+/>
+
+If at least one text field uses the 'verify on blur' strategy, do not disable the submit button. Otherwise, the mobile user will have to disable their keyboard first to have their input field verified, before they can click the submit button. The only exception is if there are required input fields.
+
+<ImageGrid
+    images={[SubmitButtonOnBlurImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={'The submit button is disabled until the user has entered something for all the required fields.'}
+/>
+
+When you have a long form (more than three text fields) with a lot of text fields and the user forgets a required field, do not auto disable the submit button -- it will take them some effort to check what they are missing. Instead, allow the user to click the submit button, and highlight the errors for them.
+
+<React.Fragment>
+    <ImageGrid
+        images={[SubmitButtonOnClickImage]}
+        gridImageProps={{ sm: 12 }}
+        caption={
+            'For a long form, do not disable the submit button. Instead, highlight the errors after the user clicks "submit".'
+        }
+    />
+</React.Fragment>
+<Divider />
+
+# Developers
+
+We recommend using the following components to build this pattern:
+
+**Angular** - Angular Material
+
+-   [Form Field](https://material.angular.io/components/form-field/overview)
+-   [Input](https://material.angular.io/components/input/overview)
+
+**React** - Material-UI
+
+-   [Text Field](https://material-ui.com/components/text-fields/#text-field)
```

Comment:
I think this goes without saying...there's not really an alternative

Reply:
Yep, just a convenient link for them.

Reply:
I would just leave it out. We've got the recommended components section on the getting started guide. If we're linking out to specific components from all of out patterns it could get crazy. Or maybe we do it as a "More Resources" kind of section with one of our action cards to link out to assorted other things.